tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -439,8 +439,9 @@ final class TestingServer(
           field.setAccessible(true)
           field.getName -> field.get(initOptions)
         }
-        .collect { case (key, Some(value)) =>
-          key -> value
+        .collect {
+          case (key, Some(value: Boolean)) => key -> value
```

Comment:
So far in `InitializationOptions` only Boolean fields shouldn't be converted to String.